File: 백준/Bronze/5342. Billing/Billing.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.HashMap;
import java.util.Map;
public class Main {
    public static void main(String[] args) throws Exception {
        Map<String,Double> sup = new HashMap<>();
        Double rlt=0.0;
        sup.put("Paper", 57.99);
        sup.put("Printer", 120.50);
        sup.put("Planners", 31.25);
        sup.put("Binders", 22.50);
        sup.put("Calendar", 10.95);
        sup.put("Notebooks", 11.20);
        sup.put("Ink", 66.95);
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String st="";
        while((st=br.readLine())!=null)
            rlt+=sup.getOrDefault(st,0.0);
        System.out.print("$"+rlt);
    }
}

File: 백준/Bronze/5342. Billing/Billing.py
import sys
input=lambda:sys.stdin.readline().rstrip("\n")
sup_lst={"Paper":57.99,"Printer":120.50,"Planners":31.25,"Binders":22.50,"Calendar":10.95,"Notebooks":11.20,"Ink":66.95}
rlt=0
while True:
    st=input()
    if st == "":
        break
    else:
        rlt+=sup_lst.get(st,0)
print(f"${rlt}")